openstack/keystone/requirements.yaml

```diff
@@ -2,15 +2,16 @@ dependencies:
   - condition: percona_cluster.enabled
     name: percona_cluster
     repository: https://charts.eu-de-2.cloud.sap
-    version: 1.0.5
+    version: 1.0.6
   - condition: mariadb.enabled
     name: mariadb
     repository: https://charts.eu-de-2.cloud.sap
     version: 0.1.16
   - name: memcached
     repository: https://charts.eu-de-2.cloud.sap
     version: 0.0.3
-  - name: mysql_metrics
+  - condition: mysql_metrics.enabled
```

Comment:
Does this require a respective value in values.yaml?

Reply:
you're absolutely right, thanks Maurice!